ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -327,15 +339,7 @@ open class BadgeView: UIView {
     open override func layoutSubviews() {
         super.layoutSubviews()
         backgroundView.frame = bounds
-
-        let labelHeight = label.font.deviceLineHeight
-        let labelSize = label.sizeThatFits(CGSize(width: CGFloat.infinity, height: CGFloat.infinity))
-        let fittingLabelWidth = UIScreen.main.roundToDevicePixels(labelSize.width)
-
-        let minLabelWidth = minWidth - size.horizontalPadding * 2
-        let maxLabelWidth = frame.width - size.horizontalPadding * 2
-        let labelWidth = max(minLabelWidth, min(maxLabelWidth, fittingLabelWidth))
-        label.frame = CGRect(x: size.horizontalPadding, y: size.verticalPadding, width: labelWidth, height: labelHeight)
```

Comment:
Honestly I can barely make heads or tails of this--I can see _what_ it's doing, but I don't understand _why_. Is there any risk to replacing this logic for, say, extremely long or extremely short labels?

Reply:
can you elaborate more for me? at least there is min width for the label so the short one it is taken care of ... but is the new logic really different from what it was doing before?

Reply:
Nope you're right, my bad. Great optimization!